docs/reference/all_strategies.rst

```diff
@@ -72,6 +72,9 @@ Here are the docstrings of all the strategies in the library.
 .. automodule:: axelrod.strategies.handshake
    :members:
    :undoc-members:
+.. automodule:: axelrod.strategies.human
```

Comment:
I don't know the answer to this question **but** should this be documented/listed in `all_strategies`? I wonder if it would be best left outside of it as that index gives a nice list of 'research' strategies (the cheaters being an immediate counter example to that point of view). No strong feelings from me just think it's worth thinking about.